joss/paper.md

```diff
@@ -35,7 +35,9 @@ bibliography: papers.bib
 
-Implementations of the Dupuit-Forcheimer model often encounter numerical instabilities as the water table intersects the surface and groundwater return flow occurs through a seepage face, due to the presence of a discontinuity in the energy gradient from inside the hillslope (where it is determined by the water table) to the seepage face (where it is determined by the topography). @marcais_dynamic_2017 introduced a regularization that smooths the transition between subsurface flow and surface flow. Although introduced for numerical stability and not based on physical principles, this smoothing may reproduce the effect of subgrid heterogeneity where saturation within a grid cell is unlikely to be homogenous or well-reproduced by a binary condition (saturated vs unsaturated).
+Groundwater models of varying complexity are available for different purposes. Fully coupled groundwater and surface water models such as PARFLOW [@kollet_integrated_2006], CATHY [@camporese_surface-subsurface_2010], and PIHM [@qu_semidiscrete_2007] solve the three-dimensional Richards equation for variably saturated flow, and couple this with precipitation and runoff components. MODFLOW [@langevin_modflow_2019] also solves the three dimensional Richards equation, and may be coupled to precipitation and runoff models, as in GSFLOW [@regan_gsflow_2018]. More parsimonious models are also available that solve the hillslope storage Boussinesq equations, which model groundwater flow in quasi-two dimensional hillslopes [@marcais_dynamic_2017, @broda_low-dimensional_2012] using the Dupuit-Forcheimer approximation. Two-dimensional implementations of the Boussinesq model are common, but do not appear to be widely available as open source packages. The simplicity and computational efficiency of this method is advantageous for capturing the first-order effects of groundwater flow on other Earth surface processes. More sophisticated groundwater models may be necessary depending on the hydrological features that the user intends to capture. 
```

Comment:
Some small thoughts on content and clarity:
* "as in GSFLOW" -> "as is implemented in GSFLOW"?
* I might define parsimonious (or use the words that define it instead of that term). My sense is that different communities have different ideas about what a parsimonious model is and how one is identified. 
* You nicely define the Richards equation and the D-F approximation. I'd recommend adding a similar statement of what exactly the Boussinesq equations are. I'd keep your statement that the Boussinesq models are implemented for gw flow in 2D hill slopes, but add what they are. You might even introduce this at the end of the prior paragraph, after the statement "free surface with groundwater flow driven by gradients in water table elevation".

Reply:
Addressed in most recent commit! Thanks :)